yoyodyne/defaults.py

```diff
@@ -60,11 +60,8 @@
 LOG_WANDB = False
 TEACHER_FORCING = True
 
-# LR Scheduler arguments.
-WARMUP_STEPS = 0
-START_FACTOR = 1 / 3
-END_FACTOR = 1.0
-TOTAL_DECAY_STEPS = 5
+# LR scheduler arguments.
+WARMUP_EPOCHS = 5
```

Comment:
Any reason for 5?

Reply:
Seemed like a good guess as any. This is giving me SOTA (?) performance in another project without any tuning...